rtorrent-flood/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '2'
+services:
+  plex:
```

Comment:
I don't like the service name "Plex", I mean Plex is good software, but it's proprietary software. This is just an aesthetic preference, I know. Maybe it would be better to keep the docker-compose.yml simple without specifying a version (3.1 is the latest) or a service name.